docs/security.html

```diff
@@ -1136,13 +1136,29 @@ <h3 class="anchor-heading"><a id="security_sasl" class="anchor-link"></a><a href
     </ol>
 
     <h3 class="anchor-heading"><a id="security_authz" class="anchor-link"></a><a href="#security_authz">7.4 Authorization and ACLs</a></h3>
-    Kafka ships with a pluggable Authorizer and an out-of-box authorizer implementation that uses zookeeper to store all the acls. The Authorizer is configured by setting <tt>authorizer.class.name</tt> in server.properties. To enable the out of the box implementation use:
+    Kafka ships with a pluggable authorization framework, which is configured by setting <tt>authorizer.class.name</tt> in server.properties. Configured implementations must extend <code>org.apache.kafka.server.authorizer.Authorizer</code>. Kafka provides default implementations which store ACLs in the cluster metadata (either Zookeeper or the KRaft metadata log).
```

Comment:
Should we mention "server.properties" explicitly? Do we do that elsewhere in the docs?

Reply:
We do, but it does seem a little odd. I tried to reword it without the explicit mention of `server.properties`.